bazel/toolchain/Dockerfile.alpine

```diff
@@ -0,0 +1,66 @@
+FROM alpine AS build
+
+RUN apk update
+RUN apk add git curl python3 clang cmake lld samurai musl-dev linux-headers
+
+WORKDIR /opt
+
+RUN git clone --branch llvmorg-16.0.6 --single-branch https://github.com/llvm/llvm-project.git
+
+ENV CC=clang
+ENV CXX=clang++
+
+RUN cmake -G Ninja -DLLVM_TARGETS_TO_BUILD='AArch64;X86' \
+  -DCMAKE_INSTALL_PREFIX=/opt/install \
+  -B /opt/build \
+  -DCMAKE_BUILD_TYPE=Release \
+  -DLLVM_BUILD_STATIC=ON \
+  -DLLVM_ENABLE_PIC=ON \
+  -DLLVM_INCLUDE_TESTS=OFF \
+  -DLLVM_INCLUDE_EXAMPLES=OFF \
+  -DLLVM_INCLUDE_UTILS=OFF \
+  -DLLVM_INCLUDE_DOCS=OFF \
+  -DLLVM_ENABLE_ZLIB=OFF \
+  -DLLVM_ENABLE_TERMINFO=OFF \
+  -DLLVM_ENABLE_ZSTD=OFF \
+  -DLLVM_ENABLE_Z3_SOLVER=OFF \
+  -DLLVM_ENABLE_DIA_SDK=OFF \
+  -DLLVM_ENABLE_LIBEDIT=OFF \
+  -DLLVM_ENABLE_LIBXML2=OFF \
+  -DLLVM_ENABLE_BACKTRACES=OFF \
+  -DLLVM_ENABLE_PROJECTS="clang;lld;lldb;clang-tools-extra" \
```

Comment:
Could you add the `llvm-bolt` project as well in case we were to ever use it? Not on the immediate roadmap so in case of any issues just ignore for now.

Reply:
Done